e2e_tests/Makefile

```diff
@@ -0,0 +1,12 @@
+NGINX_API =
+AWS_REGION = us-east-2
+PREFIX = test-runner
+TAG = edge
+PYTEST_ARGS =
+AWS_CREDENTIALS = ~/.aws/credentials
+
+build:
+	docker build -t $(PREFIX):$(TAG) -f docker/Dockerfile ..
+
+run-tests: build
```

Comment:
Should `run-tests` depend on build?

In the instructions in the README, you already asks to run `make build` command separately.

Reply:
I'll update README, run-tests should depend on the build for the pipeline automation.